Added 54x70mm label format

ceres-c opened the pull request:
Pretty straightforward, I've just added a label size. I hope this was the right way to go

--edit--
I guess this is not necessary, as one can add a custom label size from the web UI. Feel free to close this PR if you deem it unnecessary, sorry

michaelrsweet commented on PKGBUILD:
```diff
@@ -0,0 +1,40 @@
+# Maintainer: WorMzy Tykashi <wormzy.tykashi@gmail.com>
```
I don't think this file belongs in the PR.

ceres-c replied:
Ah damn, I made a mess

michaelrsweet commented on static-resources/lprint-de-strings.h:
```diff
@@ -85,6 +85,7 @@ static const char * const lprint_de_strings =
 "\"media.oe_file-folder-label_0.5625x3.4375in\" = \"9/16 x 3-7/16 File Folder Label\";\n"
 "\"media.oe_hanging-file-tab-insert_0.5625x2in\" = \"9/16 x 2 hängende Datei Tab Einfügen Etikett\";\n"
 "\"media.oe_internet-postage-confirmation-label_2.3125x10.5in\" = \"2-5/16 x 10-1/2 Internet Postage Confirmation Label\";\n"
+"\"om_diskette-label_54x70mm\" = \"54 x 70mm Disketten Etikett\";\n"
```
These actually need to be in the original .strings files.

michaelrsweet reviewed (changes requested):
I have no objection to adding support for this label size, note the comments about the PKGBUILD file and the additions to the strings headers that should be in the original strings files (also).

ceres-c commented:
Added to .strings files and removed PKGBUILD (daily reminder not to `git add .` what you did the previous day at 3AM).
Should I leave the change in the C file or is it better to get rid of that as well?

michaelrsweet commented:
> Should I leave the change in the C file or is it better to get rid of that as well?

The C file (lprint-dymo.c) is needed for the media size to show up, so that should stay... :)

michaelrsweet reviewed (approved):
Updates look good!

ceres-c commented:
> The C file (lprint-dymo.c) is needed for the media size to show up, so that should stay... :)

Good, I noticed today there were some media formats in the strings file that were not listed in the .c so I was wondering if there was some redundancy somewhere. Didn't have the printer setup at hand to test the available formats
